pyproject.toml

```diff
@@ -33,6 +33,7 @@ pytest-cov = "^4.0.0"
 openmock = "^2.3.0"
 requests-mock = "^1.11.0"
 freezegun = "^1.2.2"
+psycopg2 = "^2.9.3"
```

Comment:
Agreed with Greg, if this is only used in the lambda, please move it to `lambda-dev`. If you're installing with requirements.txt then you shouldn't need to do anything else, except run pre-commit locally to update your local version.

Reply:
Correct me if I am wrong, but the Dockerized lambdas require a Dockerfile to exist within their same directory. So we need to revisit whether or not we want to use Dockerized lambdas or the regular lambda_alpha_.PythonFunction.

Here is the ChatGPT Conclusion:

Choose AWS Lambda's Built-in Runtimes if you want simplicity, ease of use, and are working within the constraints of the supported languages and package size limits.

Opt for Dockerized Lambdas if you need greater control over the runtime environment, have large dependencies, or use languages/runtimes not natively supported by AWS Lambda.

Reply:
Either way you'll need this in the requirements file that is used to build the lambdas because either the Dockerfile will need to install the dependencies or the PythonLambdaAlpha will need to install them.

PythonLambdaAlpha is just a nice wrapper on the ContainerLambda for you so all you need is a `requirements.txt` file and it will create the Dockerfile automagically for you under the hood and build the container image and push it up to an ECR for you.